### PR TITLE
links are copy-able

### PR DESCRIPTION
**Pull request by peopledrivemecrazy**
base.css -> a tags are copy-able.
GuideContents.svelte -> a tags on sidebar will need to remain not copy-able.

please refer #31 

Also added transition properties to other generic browers.

### CODE REVIEW

File: base.css [hunk 1]
```diff
@@ -477,6 +476,10 @@ input[type="checkbox"]::after {
 	background: white;
 	box-shadow: 0 0px 1px rgba(0,0,0,.4), 0 4px 2px rgba(0,0,0,.1);
 	-webkit-transition: background .2s ease-out, left .2s ease-out;
+	-moz-transition: background .2s ease-out, left .2s ease-out;
```
Comment (benmccann) [inline base.css:479]: is it necessary to use the browser-specific versions still?

MDN say `-webkit-transition` is deprecated: https://developer.mozilla.org/en-US/docs/Web/CSS/@media/-webkit-transition

Comment (peopledrivemecrazy) [reply base.css:479]: @benmccann thank you for reviewing this. 

Originally site-kit has `-webkit-transition` and visual studio code was reporting lint warnings and I went ahead and added all vendor prefix. 

PS.  just adding `transition` stops lint warnings too.

Comment (peopledrivemecrazy) [reply base.css:479]: Also I would be happy to revert them just how it was before

Comment (benmccann) [reply base.css:479]: My question is can you put just `transition` and remove the others?

Comment (peopledrivemecrazy) [reply base.css]: I just added the `transition`

**Review by benmccann — APPROVED** [review]

**Comment by benmccann** [comment]
FYI, we're moving this repo to https://github.com/sveltejs/sites/tree/master/packages/site-kit. That location is currently used only for the SvelteKit docs, but we hope to eventually use it for all docs

**Comment by peopledrivemecrazy** [comment]
Closing due to https://github.com/sveltejs/sites/pull/44

**Pull request closed without merging**